Code completion in completion1.xml for layout_width=^"fill_parent" selecting "wrap_content":
@@ -8 +8
-         android:layout_width=^"fill_parent"
+         android:layout_width="wrap_content"^
